packages/langium/src/utils/uri-utils.ts

```diff
@@ -36,6 +36,10 @@ export namespace UriUtils {
             if (toParts[0] && upperCaseDriveLetter.test(toParts[0])) {
                 toParts[0] = toParts[0].toLowerCase();
             }
+            if (fromParts[0] !== toParts[0]) {
+                // in case of different drive letters, we cannot compute a relative path, so...
+                return toPath.toLowerCase();
```

Comment:
Question: Why lowercase the whole string? We can just keep the original capitalization.

Reply:
True.